setup.py

```diff
@@ -0,0 +1,28 @@
+import setuptools
+from setuptools import find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+setuptools.setup(
+    name="tetramer_validator",
+    version="0.0.1",
+    author="IEDB",
+    author_email="help@iedb.org",
+    description="A small package to validate tetramers",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    # url="https://github.com/IEDB/tetramer-validator",
```

Comment:
This should not be a comment. It should either be a proper value (uncommented), or the line should be deleted.

Reply:
```suggestion
    url="https://github.com/IEDB/tetramer-validator",
```